### PR TITLE
Update CAPI single paidFor

### PR DESCRIPTION
**Pull request by mxdvl**
## What does this change?

Update the `capi-single-paidfor` template on US’s request.

Compared to the previous version, the image is more prominent, and the badge link is removed.

## How to test

`npm run preview`, or via an `adTest` param and GAM.

## How can we measure success?

Better engagement.

## Images

capi-single-paidfor template

### CODE REVIEW

File: src/capi-single-paidfor/web/index.js [hunk 1]
```diff
@@ -59,25 +53,18 @@ function populateCard(host, responseJson) {
...
-      ${arrowRight}
-    </a>
-    </div>
     <div class="badge js-badge">
       Paid for by
       <a class="badge__link" href="${clickMacro}${responseJson.branding.logo.link}" data-link-name="badge">
         <img class="badge__logo" src="${getValue('[%BrandLogo%]', responseJson.branding.logo.src)}" alt="">
       </a>
+    </div>
     </div>`;
```
Comment (arelra) [inline src/capi-single-paidfor/web/index.js:69]: Minor if you want to - can we improve the HTML indentation for this block?

Comment (mxdvl) [reply src/capi-single-paidfor/web/index.js:69]: nah

Comment (mxdvl) [reply src/capi-single-paidfor/web/index.js:69]: Go on, then…

File: src/capi-single-paidfor/web/index.scss [hunk 2]
```diff
@@ -1,15 +1,57 @@
...
+      flex-grow: 1;
+  }
+
+  .badge {
+    width: 200px;
+    text-align: left;
+
+    &::before {
+      display: none;
+    }
```
Comment (arelra) [inline src/capi-single-paidfor/web/index.scss:45]: This seems a bit weird? Or am I missing some CSS trick?

Comment (mxdvl) [reply src/capi-single-paidfor/web/index.scss:45]: You _are_ missing a trick: it's SASS.

The `&` is replaced by the parent item when converted to CSS. Here we could simplify it if we're not adding any other rule…

Comment (arelra) [reply src/capi-single-paidfor/web/index.scss:45]: Yeah I understand its SASS but why the `display:none` on `::before` ?

Comment (mxdvl) [reply src/capi-single-paidfor/web/index.scss:45]: It's overriding a rule that displays a line…

**Review by zekehuntergreen — APPROVED** [review]
👍

**Pull request merged**